### PR TITLE
topology: hda: remove virtual DAPM routes

### PR DESCRIPTION
**Pull request by ranj063**
These are not really required to suppress the machine driver errors.

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

File: tools/topology/m4/utils.m4 [hunk 1]
```diff
@@ -101,33 +101,21 @@ define(`W_DATA',
 `}')
 
 dnl VIRTUAL_DAPM_ROUTE_OUT(name, dai type, dai index, direction, index)
+dnl No routes needed. Macro retained for simplicity
```
Comment (plbossart) [inline tools/topology/m4/utils.m4:104]: Why not remove the macro then if it's not useful?

Comment (ranj063) [reply tools/topology/m4/utils.m4:104]: i'd have to modify 4-5 tplg files to remove the macro. this way the change is one place and anyway we'll transition to 2.0 soon. so this wont be an issue

**Comment by lgirdwood** [comment]
@ranj063 some conflicts.

**Comment by ranj063** [comment]
> @ranj063 some conflicts.

fixed @lgirdwood

**Pull request merged**